migrations/sql/V2019.2019.12.29.3.33__add_application_progress.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE IF NOT EXISTS application_progress_type (
```

Comment:
I think application_progress_status is more representative, should be a quick change to make. Can you also please add table comments?

Reply:
Will do!